simulator/hart.h

```diff
@@ -25,6 +27,8 @@ class Hart final {
     interpreter::Fetch fetch_;
     interpreter::Decoder decoder_;
     interpreter::Executor executor_;
+    static inline constexpr const size_t BB_CACHE_SIZE = 256;
```

Comment:
Seems overcomplex for me, @Mirageinvo pls check and give ur opinion

Reply:
Discussed in chat